font-types/src/tag.rs

```diff
@@ -83,10 +81,15 @@ impl Tag {
     pub const fn from_be_bytes(bytes: [u8; 4]) -> Self {
         Self(bytes)
     }
+
+    /// Return the raw byte array representing this tag.
+    pub fn to_bytes(self) -> [u8; 4] {
```

Comment:
Maybe this isn't a convention in Rust but to_bytes sounds to me like it does a conversion whereas this feels more like as_bytes

Reply:
There is a handy chart for deciding when to use `to_`, `as_`, and `into_`: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

..which is slightly confusing, because it feels like we should use `into`, but I don't generally see `into` on Copy types? But I'll go with that anyway.

`as_bytes` would make sense if the signature was `(&self) -> &[u8]`